ocpmodels/common/utils.py

```diff
@@ -74,7 +74,12 @@ def warmup_lr_lambda(current_epoch, optim_config):
     """Returns a learning rate multiplier.
     Till `warmup_epochs`, learning rate linearly increases to `initial_lr`,
     and then gets multiplied by `lr_gamma` every time a milestone is crossed.
+    Specifying `lr_schedule`: "fixed" supports a fixed learning schedule.
     """
+    warmup_option = optim_config.get("lr_schedule", "")
+    if warmup_option == "fixed":
+        return 1.0
+
```

Comment:
Could we handle parsing of this arg elsewhere (e.g. in the trainer?)? It's slightly odd for the `warmup_lr_lambda` scheduler to return a fixed lr multiplier.

Reply:
Yeah, it's better suited for a separate PR.

Reply:
Let me remove it from here and we can handle LR scheduling separately.